src/layout/AppLayout/AppLayoutStyles.ts

```diff
@@ -0,0 +1,10 @@
+import { styled } from "@mui/material/styles";
+import { Box } from "@mui/material";
+
+export const AppContainer = styled(Box)({
```

Comment:
Just for simple styling use sx property in Box.

Reply:
got it